src/marketplace/ListingsView.js

```diff
@@ -10,15 +10,19 @@ type Props = {|
 |};
 type State = {|
 	userId: string,
+	//This allows for all the items that are for sale to be with in one array
+	itemsForSale: Array<Object>
```

Comment:
Thanks for putting it in the state, but it isn't just an array of any type of object. You are making some assumptions about what will be in this array, so those should be indicated to other people in the Array<whateverTheTypeShouldBe> part

Reply:
Did it with a interface